braidpool-primitives/src/braid/mod.rs

```diff
@@ -112,66 +157,23 @@ impl Braid {
         cohorts
```

Comment:
Call braid.cohorts() here with the parents HashMap (and cached children hashmap)